lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,15 @@ div.course-wrapper {
       }
     }
 
+    .required-content-msg {
```

Comment:
I might use something like ".sequential-status-message" to make clear the kind of message this is referring to.